README.md

```diff
@@ -76,37 +75,97 @@ let dog = new Dog("Spot");
-to build on top of like when using a classic programming language model. As the popular
-phrase goes: "With power comes great responsibility." It can be tempting to overuse these
-features. When creating "child" `class`es, we defer to _subtyping_ relation principles
-called _(strong) behavioral subtyping_ in programming language theory. 
-
-- You probably won't use inheritance often, but as your code bases get larger, you are more
+You probably won't use inheritance often, but as your code bases get larger, you are more
 likely to find a need for it. If you're creating a number of objects that have similar
 features, then creating a generic object type to contain all the shared functionality,
 and inheriting those features in more specialized objects can be convenient and useful.
 
-- When creating a parent-child `class` relation, it is _semantic_ rather than merely
-_syntactic_. Let's use `Vehicle` as a parent `class` with `Car` as a child
-`class` of `Vehicle` as an example. In _classical_ inheritance, it creates a _copy_ of the
-behavior from parent. However, in JavaScript when we create the object it does not copy the
-properties or behavior, it creates a _link_. `class`es should _never_ be extended unless it
-is for the sake of compatibility with newer JavaScript features. This is due to the way
-inheritance works on top of vanilla JavaScript code.
+It is recommended that `class`es should _never_ be extended unless it is for the sake of
+compatibility with newer JavaScript features, like building UI components frameworks in React
+and Angular. This is due to the way inheritance works on top of vanilla JavaScript code, and
+could create a multitude of unforseen headaches down the road. Always start with the simplest
+solution and progress to more complex solutions only as-needed.
+
+### JavaScript Inheritance is Not Like Other Languages
+
+JavaScript can be a bit confusing since it is a dynamic language that uses syntactic
+sugar to leverage `class` implementations and inheritance similar to classic programming
+languages like C++, Java, or PHP. However, [JavaScript's inheritance model](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Details_of_the_Object_Model)
+is quite powerful to build on top of like when using a class-based (or sometimes called classical)
+programming language model. As the popular phrase goes: "With power comes great responsibility."
+It can be tempting to overuse these features.
+
+In Ruby, a parent `class` is an object, but it is neither an instance of itself nor of an object.
+The parent `class` object has significantly more methods than an instance of the parent. In Ruby,
+the parent is a _special_ type of object that includes `class` methods, including `#new`and
+`#initialize` which are needed to create instances of itself. In Javascript, there are many ways
+to create objects.
+
+In _classical_ inheritance, it creates a _copy_ of the behavior from parent. However, in
```

Comment:
I think it may be ambiguous here

Reply:
Going to add a link out to another resource here. I think diving too deep into the differences isn't entirely necessary for this readme.

Reply:
agreed